packages/blade/src/components/Typography/BaseText/types.ts

```diff
@@ -64,7 +64,7 @@ export type BaseTextProps = {
    * React Native only
    */
   numberOfLines?: number;
-  componentName?: 'base-text' | 'text' | 'title' | 'heading' | 'code';
```

Comment:
@snitin315 can you check with design that do we need to update this in our amount components? because amount component is basically permutations and combinations of all the typography elements